docs/src/pages/developers/technical-faq/index.mdx

```diff
@@ -121,15 +121,7 @@ If you're encountering this issue in production mode, try to check the worker lo
 
 ## How can I make imports run faster?
 
-The included `docker-compose` configuration for production mode has no understanding of how powerful your server is. It will run just a single instance of the services involved in data imports, `worker` , `ingest-file` and `convert-document`.
-
-The easiest way to speed up processing is to scale up those services. Make a shell script to start docker-compose with a set of arguments like this:
-
-```bash
-docker-compose up --scale ingest-file=8 --scale convert-document=4 --scale worker=2
-```
-
-The number of `ingest-file` processes could be the number of CPUs in your machine, and `convert-document` needs to be scaled up for imports with many office documents, but never higher than `ingest-file`.
```

Comment:
Suggestion:

```
The number of `ingest-file` processes could be the number of CPUs in your machine. Since performance and scaling depends a lot on types of workloads you are seeing you may need to review and adjust these settings accordingly.
```

followed by what you wrote:

```
Please also see the [Scaling Workers](/developers/installation.mdx#scaling-workers) section in the installation guide.
```